USER_MANUAL.md

```diff
@@ -528,5 +528,48 @@ The late connecing client has validated the local state before sending the "Pass
   * In the Visual Logger click `Load` and load in `first.vlog` and `second.vlog` simultaneously, using the shift key to select both files.
   * The files should successfully load and display expected logs. The contents of two log files should be offset by the time delta between the two recordings.
 
+##### Gameplay Debugger gym
+* Test that:
+  * The Gameplay Debugger works like native Unreal with spatial networking disabled.
+  * The Gameplay Debugger can be toggled between player and manual server tracking modes.
+  * The Gameplay Debugger can be cycled through servers in manual server tracking mode.
+  * The Gameplay Debugger hides server cycling key prompts whilst in player tracking mode.
+  * The Gameplay Debugger tracks player movements across worker boundaries in player tracking mode.
+  * The Gameplay Debugger does not track player movements across worker boundaries in manual tracking mode.
+  * The Gameplay Debugger displays debug information accurately.
+* How to test:
+  * You will need a numpad on your keyboard to test this. If you don't have one, you can use the windows onscreen keyboard.
+  * Open `Content\Maps\GameplayDebugger\GameplayDebuggerManualTest.umap`
+  * Click the `Play` button in the Unreal toolbar to start PIE session, uncheck the option for `SpatialOS networking`, and set `Number of players` to one.
+  * Observe that the `GameplayDebuggerGymCube` moves forwards and backwards between the two servers.
+  * While the camera is looking directly at the cube, press `'`(Apostrophe) to enable the gameplay debugger on the cube.
+  * Observe in the gameplay debugger status panel that `Debug Actor` is set to `GameplayDebuggerGymCube`. If it is not, toggle the gameplay debugger off by pressing `'` and try again.
+  * Observe there is no mention of the spatial multi worker specific additions for `server tracking modes` or `selecting servers` in the gameplay debugger status panel.
+  * Press `numpad4` to enable the gameplay debugger EQS output.
+  * Observe there are "EQS debug spheres" drawn at regular intervals around the cube as it moves along its path.
+  * Click the `Stop` (play) button in the Unreal toolbar to stop the PIE session
```

Comment:
```suggestion
  * Click the `Stop` button in the Unreal toolbar to stop the PIE session
```

Reply:
I find this addition more confusing than clarifying, the mention of PIE is enough.